src/box/sql/vdbe.c

```diff
@@ -4340,7 +4340,7 @@ case OP_IncMaxid: {
 	assert(pOp->p1 > 0);
 	pOut = vdbe_prepare_null_out(p, pOp->p1);
 	uint64_t u;
-	if (tarantoolsqlIncrementMaxid(&u) != 0)
+	if (box_generate_space_id(&u) != 0)
```

Comment:
I think you should also change the op code name, because `OP_IncMaxid` doesn't reflect what this operation actually does.